src/main/java/frc/robot/PowerController.java

```diff
@@ -0,0 +1,168 @@
+package frc.robot;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import edu.wpi.first.wpilibj.PowerDistribution;
+import frc.robot.subsystems.tank.TankSubsystem;
+import static frc.robot.Constants.TankConstants.*;
+
+public class PowerController {
+
+    private static PowerDistribution PDP = new PowerDistribution();
+
+    private ControllableSubsystem tankSubsystem;
+
+//    private ControllableSubsystem climbSubsystem;
+
+   // private ControllableSubsystem intakeSubsystem;
+
+   // private ControllableSubsystem shooterSubsystem;
+
+    private int currentCurrentLimit;
```

Comment:
This is a personal style recommendation: when I name variables that correspond to physical things I like to append the units to the variable name. In this case it would be `currentCurrentLimit_A`. This way I can check units by looking at variable names.